cerebrium/fine-tuning/auto-deploying.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Auto-Deploying your Trained Model
+description: Learn how to seamlessly deploy your trained model to a production environment.
+---
+
+# Introduction
+
+If you are reading this guide, chances are that you have successfully trained a model on Cerebrium. Congratulations!
+
+Now that your model has been successfully trained, you have two options for deploying to a production environment:
+
+- The first is to download the trained adapter weights using `cerebrium download-model` and write your own deployment code.
+- The second is to use the built-in auto-deployment feature of Cerebrium's trainer.
+
+While you have more flexibility writing custom deployment code for your finetuned weights, the second option is the method that we recommend for most users!  
+It's faster, easier and gets your model up and running in no time.
+
+Auto-deployment streamlines the process of getting your model into a production environment. With this approach, deployment occurs automatically as soon as training is completed. This feature not only saves time but also provides a seamless way to test your model's performance before considering any modifications to the deployment code.
```

Comment:
You can get right of this line. You have mentioned 3 times how seamless and fast autodeployment is - no fluf